tests/unittests/test_netinfo.py

```diff
@@ -21,19 +25,15 @@
 FREEBSD_NETDEV_OUT = readResource("netinfo/freebsd-netdev-formatted-output")
 
 
-class TestNetInfo(CiTestCase):
-
-    maxDiff = None
-    with_logs = True
-
+class TestNetInfo:
```

Comment:
Removed base class from these tests. Changes in this file that are related to the changed are the newly parametrized ones.